src/main/java/TestBase.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.concurrent.TimeUnit;
+
+public class TestBase {
+    public static WebDriver driver;
```

Comment:
Why static? It is bad idea, because it will produce problems in multi-threading solutions.